doc/modules/neural_networks_supervised.rst

```diff
@@ -146,7 +146,8 @@ See the examples below and the docstring of
 .. topic:: Examples:
 
  * :ref:`sphx_glr_auto_examples_neural_networks_plot_mlp_training_curves.py`
- * :ref:`sphx_glr_auto_examples_neural_networks_plot_mnist_filters.py`
+ * See :ref:`sphx_glr_auto_examples_neural_networks_plot_mnist_filters.py` for
+   visualization of varying regularization in MLP.
```

Comment:
this example doesn't really demonstrate changing regularization, it only shows how to visualize trained weights.

Reply:
so would a appropriate text be `See "link here" for visualized representation of trained weights`?

Reply:
that makes sense.